fields/class-swp-acf-color-palette-v5.php

```diff
@@ -195,33 +195,17 @@ function render_field( $field ) {
 		 * @return n/a
 		 */
 		function input_admin_enqueue_scripts() {
-			global $wp_scripts;
-
 			$url = $this->settings['url'];
 			$version = $this->settings['version'];
 			$min = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
-			// register if not already (on front end)
-			// http://wordpress.stackexchange.com/questions/82718/how-do-i-implement-the-wordpress-iris-picker-into-my-plugin-on-the-front-end
-			if ( ! isset( $wp_scripts->registered['iris'] ) ) {
-				wp_register_style( 'wp-color-picker', admin_url( 'css/color-picker.css' ), array(), '', true );
-
-				wp_register_script( 'iris', admin_url( 'js/iris.min.js' ), array( 'jquery-ui-draggable', 'jquery-ui-slider', 'jquery-touch-punch' ), '1.0.7', true );
-				wp_register_script( 'wp-color-picker', admin_url( 'js/color-picker.min.js' ), array( 'iris' ), '', true );
-
-			    wp_localize_script( 'wp-color-picker', 'wpColorPickerL10n', array(
-			        'clear'			=> __( 'Clear', 'swp-acf-cp' ),
-			        'defaultString'	=> __( 'Default', 'swp-acf-cp' ),
-			        'pick'			=> __( 'Select Color', 'swp-acf-cp' ),
-			        'current'		=> __( 'Current Color', 'swp-acf-cp' )
-			    ) );
-
-			}
```

Comment:
Can you restore this part please? That is no relation with the new way to load wp-color-picker-alpha ;)